node/tests/RedisClusterClient.test.ts

```diff
@@ -307,7 +307,6 @@ describe("GlideClusterClient", () => {
                 client.smove("abc", "zxy", "value"),
```

Comment:
I noticed zdiff and zdiffstore were added in 6.2.0, do we also want to add an "if (!versionLessThan62)" block for them?